packages/bee-q/src/components/breadcrumb/bq-breadcrumb.tsx

```diff
@@ -101,7 +101,7 @@ export class BqBreadcrumb {
 
   render() {
     return (
-      <nav class="flex items-center" role="list" part="navigation">
+      <nav class="flex items-center" role="list" aria-label="breadcrumbs" part="navigation">
```

Comment:
I think the value of this `aria-label` should be a property of `bq-breadcrumb`, what do you think?